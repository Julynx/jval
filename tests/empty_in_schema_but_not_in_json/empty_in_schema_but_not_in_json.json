{
    "role": "user",
    "name": "Alice",
    "cars": ["kia", "bmw"]
}