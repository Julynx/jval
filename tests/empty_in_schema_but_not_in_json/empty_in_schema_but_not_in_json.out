{
    "role": "user",
    "phone": "123456789",
    "name": "Alice",
    "cars": ["kia", "bmw"]
}